body.tex

```diff
@@ -692,7 +699,95 @@ \subsection{Reporting Cycle}
+This June period of post-sandbox spending plan development and cost calculation involves the Directors Office staff making updates to the planning tools (Appendix~\ref{sec:planning-tools}), with review by teh departmental management teams.
+
+The changes to the spending plan, once captured in the Rubin planning tools, are implemented in NOIRLab's and SLAC's lab-wide budget models in July.
+Statements of work for the next fiscal year's contracts and MPOs are straightforwardly derived from the scrubbed plans, for their July deadlines.
+
+The teams then plan work against their minor milestones in September, incorporating any community input they may have collected at the annual project and community workshop (PCW) in August.
+
+\begin{figure}[!ht]
+\includegraphics[width=0.95\linewidth]{figures/annual-budget-cycle.png}
+\caption{Annual budget planning cycle.}
+\label{fig:annual-cycle}
+\end{figure}
+
+
+\subsection{Risk-aware budgeting and Requests Beyond Target}
+
+The ground-up spending plan developed during the annual cycle is to cover operations work defined by the milestone activities, as well as any regular work captured in bucket epics.
+This work includes mitigation of risks held by the teams, as described in the response plans in the risk register.
+Any risks that are realized during the year will require action that is not budgeted for.
+Additional spending to address realized risks is enabled by Rubin's request beyond target (RBT) process, whereby reserve funds are released for use by the departments in response to a specific mid-year request made in a custom Jira ticket.
+
+Standard terminology for financial reserves is that ``management reserve'' describes funds held back to pay for the unknown unknowns, while ``contingency reserve'' is used for known unknowns (i.e.\ the risks in the risk register).
+Rubin Operations' two funding agencies offer differing guidelines on how reserve should be managed.
+DOE allows reserves to be built up and carried forward at SLAC, and used as both contingency and management reserve.
+DOE HEP holds additional reserves centrally, which can also be used as both contingency and management reserve.
+NSF AST allows line item budget over-estimation by up to 10%, to allow for increases in cost.
+These generate carry-forward (along with any other under-spending that may occur), which we use as contingency reserve.
+NSF also holds reserves of both kinds centrally, to cover high cost impact realized risks as well as unexpected events.
+
+The cost impact analysis of the Rubin Operations risk register informs the level of contingency reserve needed.
+Residual (post-mitigation) cost exposure for each risk is computed by taking the  product of the residual probability of the risk being realized (in any given year) and the likely annual cost of addressing that risk.
```

Comment:
maybe add "While Rubin holds a number of potentially very costly risks, these are generally very unlikely to occur."

Reply:
Good. I looked at the risk analysis: it's the "significant" risks that carry most of the total cost exposure, the "damaging/major" and "catastrophic/extreme" risks don't carry much exposure at all (because their likelihood is so low, thankfully).